EIPS/eip-micropayments_standard_for_nfts_and_multi_tokens.md

```diff
@@ -0,0 +1,310 @@
+---
+eip: <to be assigned>
+title: Micropayments Standard for NFTs and Multi Tokens
+description: A standard interface for tip tokens that allows tipping to holders of NFTs and multi tokens
+author: Jules Lai (@julesl23), aka Julian Bushell
```

Comment:
```suggestion
author: Jules Lai (@julesl23)
```

Reply:
Alternatively you can do `Julian Bushell (@jules123)`.  It is the author's choice what name they wish to go by.